source/documentation/user-journeys.html.md.erb

```diff
@@ -120,6 +120,67 @@ Traders cannot remove a locked record from their TGP. The record must be in an u
 
 Traders cannot restore a record once they remove it from TGP.
 
+## Request accreditation
+
+HMRC offers advice to confirm if the trader has the correct commodity code in a goods record. To ask HMRC for advice on a record, you can use the 'Request Accreditation' endpoint.
+
+### Making a request
+
+Traders can only request advice on an existing goods record. Use the ‘Create TGP record’ endpoint to create a new record.
+
+You will need to provide all the details of the goods record in your request to HMRC. This will help the caseworker make an informed decision.
```

Comment:
This isn't true, they only need to provide the recordId, we use that to get the details of the record for case management

Reply:
Content updated